Tests/UnitTests/Identity/CustomerInfoManagerTests.swift

```diff
@@ -29,10 +29,11 @@ class BaseCustomerInfoManagerTests: TestCase {
             "subscriber": [
                 "original_app_user_id": Self.appUserID,
                 "first_seen": "2019-06-17T16:05:33Z",
-                "subscriptions": [:],
-                "other_purchases": [:],
+                "subscriptions": [:]  as [String: Any],
+                "other_purchases": [:]  as [String: Any],
```

Comment:
Seems like we have a bunch like these in this PR?
```suggestion
                "subscriptions": [:] as [String: Any],
                "other_purchases": [:] as [String: Any],
```

Reply:
Fixed, thanks!